shell/platform/linux/fl_standard_message_codec.cc

```diff
@@ -420,6 +420,10 @@ static GBytes* fl_standard_message_codec_encode_message(FlMessageCodec* codec,
 static FlValue* fl_standard_message_codec_decode_message(FlMessageCodec* codec,
                                                          GBytes* message,
                                                          GError** error) {
+  if (g_bytes_get_size(message) == 0) {
+    return fl_value_new_null();
```

Comment:
@cbracken Null messages strike again :P

How do you think we should handle null messages here: null data, or a value whose value is null? IIRC you found that this was inconsistent in https://github.com/flutter/engine/pull/25064 but I forget whether you cleaned it up later.

Reply:
You have to return the FlValue object here, because returning NULL implies there was an error (GObject convention).